specs/src/README.md

```diff
@@ -15,3 +15,4 @@
   - [mint](../../x/mint/README.md)
   - [paramfilter](../../x/paramfilter/README.md)
   - [upgrade](../../x/upgrade/README.md)
+  - [tokenfilter](../../x/tokenfilter/README.md)
```

Comment:
I suspect we also want to add it after:

1. https://github.com/celestiaorg/celestia-app/blob/766b8a5c212e99915f5689beca8bde71392cb21c/specs/src/SUMMARY.md?plain=1#L19
2. https://github.com/celestiaorg/celestia-app/blob/766b8a5c212e99915f5689beca8bde71392cb21c/specs/src/specs/state_machine_modules.md?plain=1#L11